azure-devops/provision-image.ps1

```diff
@@ -388,20 +290,11 @@ Add-MpPreference -ExclusionProcess clang-cl.exe
 Add-MpPreference -ExclusionProcess cl.exe
 Add-MpPreference -ExclusionProcess link.exe
 Add-MpPreference -ExclusionProcess python.exe
+Add-MpPreference -ExclusionProcess test.exe
 
-InstallMSI 'CMake' $CMakeUrl
-InstallZip 'Ninja' $NinjaUrl 'C:\Program Files\CMake\bin'
-InstallLLVM $LlvmUrl
 InstallPython $PythonUrl
 InstallVisualStudio -Workloads $Workloads -BootstrapperUrl $VisualStudioBootstrapperUrl
 InstallCuda -Url $CudaUrl -Features $CudaFeatures
 
-Write-Host 'Updating PATH...'
-$environmentKey = Get-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment' -Name Path
-$Env:PATH="$($environmentKey.Path);C:\Program Files\CMake\bin;C:\Program Files\LLVM\bin"
-Set-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment' `
-  -Name Path `
-  -Value "$Env:PATH"

```

Comment:
This block serves a dual purpose: it adds cmake and llvm to the local path and system path - which we no longer need since we no longer use external cmake/llvm - _and_ it picks up any other additions (*cough* python) to the system path and applies them to the local path. Removing this block entirely could be the cause of `psutil` not getting installed.

Reply:
Aha, thanks - the second purpose was completely invisible to me.